internal/plans/objchange/compatible.go

```diff
@@ -202,13 +202,13 @@ func assertObjectCompatible(schema *configschema.Block, planned, actual cty.Valu
 //
 // This function strips marks from its inputs, as they are not considered
 // relevant by the call site.
-func AssertValueCompatible(planned, actual cty.Value) []error {
+func AssertValueCompatible(planned, actual cty.Value, writeOnly bool) []error {
```

Comment:
Probably worth checking if we actually need to check write-only in `AssertObjectCompatible`, since that is used to compare what the provider planned with what the provider applied, and neither of those values should still contain write-only attributes.

Reply:
Yeah, I think you are right, that is now solved in a different place, I just got rid of the entire commit 👍